openage/convert/main.py

```diff
@@ -35,8 +36,12 @@ def convert_assets(assets, args, srcdir=None, prev_source_dir_path=None):
     if srcdir is None:
         srcdir = acquire_conversion_source_dir(prev_source_dir_path)
 
+    # Initialize game versions data
+    auxiliary_files_dir = cfg / "converter" / "games"
+    args.avail_game_eds, args.avail_game_exps = create_version_objects(auxiliary_files_dir)
+
     # Acquire game version info
-    game_version = get_game_version(srcdir)
+    game_version = get_game_version(srcdir, args)
```

Comment:
You do not need to pass all of `args` here. Only the list of game editions and the list of game expansions needs to be passed.